src/main/java/com/larscheidschmitzhermes/nexus3/github/oauth/plugin/api/GithubApiClient.java

```diff
@@ -116,7 +117,9 @@ private GithubUser retrieveGithubUser(String loginName, char[] token) throws Git
 
     private void checkUserInOrg(String githubOrg, char[] token) throws GithubAuthenticationException {
         Set<GithubOrg> orgs = getAndSerializeCollection(configuration.getGithubUserOrgsUri(), token, GithubOrg.class);
-        if (orgs.stream().noneMatch(org -> githubOrg.equals(org.getLogin()))) {
+        String[] githubOrgs = githubOrg.split(",");
+
+        if (orgs.stream().noneMatch(org -> Arrays.asList(githubOrgs).contains(org.getLogin()))) {
             throw new GithubAuthenticationException("Given username not in Organization '" + githubOrg + "'!");
```

Comment:
actually, this is more like "Given username not in org or org not allowed"

Reply:
Good idea! 👍